tests/test_image_processing_common.py

```diff
@@ -238,6 +239,52 @@ def test_image_processor_save_load_with_autoimageprocessor(self):
 
             self.assertEqual(image_processor_second.to_dict(), image_processor_first.to_dict())
 
+    def test_save_load_fast_slow(self):
```

Comment:
Added tests to make sure we can correctly cross-load fast and slow image processors directly with its class and with the auto class.